### PR TITLE
Add new driver permissions for vmware-vsphere-csi-operator

### PR DESCRIPTION
**Pull request by bertinatto**
CC @openshift/storage

### CODE REVIEW

**Comment by bertinatto** [comment]
/hold
until I manually test this

**Comment by bertinatto** [comment]
/hold cancel
I'm unable to test this at the moment because I wasn't able to get a working cluster.

File: assets/csidriveroperators/vsphere/06_clusterrole.yaml [hunk 1]
```diff
@@ -276,18 +276,13 @@ rules:
...
-  - cnsvspherevolumemigrations
+  - '*'
   verbs:
-  - create
-  - get
-  - list
-  - watch
-  - update
-  - delete
+  - '*'
```
Comment (gnufied) [inline assets/csidriveroperators/vsphere/06_clusterrole.yaml:285]: This is for enabling webhook right?

Comment (bertinatto) [reply assets/csidriveroperators/vsphere/06_clusterrole.yaml:285]: Correct, to make sure the webhook can start we need to enable the CSI migration in the driver; and once we do that the driver starts meddling with objects from this api group.

**Comment by bertinatto** [comment]
It seems like previous failures in `e2e-aws-upgrade` job fixed by https://github.com/openshift/installer/pull/5100, however, recent failures are caused by fired alerts that aren't related to this PR.

**Pull request merged**